Week-3/Session-9/Arrays/script.js

```diff
@@ -0,0 +1,9 @@
+const divideArray = (numbers) => {
+    console.log('Even numbers:')
+    const even = numbers.filter((e) => e % 2 == 0).sort().forEach((e) => console.log(e));
```

Comment:
Always pass in the callback function for ascending or descending in the sort() method so as you dont get unexpected results at times.

If callbacks are not passed in the sort(), JS tends to sometimes sort the elements in an absurd way.

Reply:
@pradyum98 
Please take a look at this comment.

Reply:
Noted @kamalmohit